### PR TITLE
Apicall

### PR DESCRIPTION
**Pull request by tta2yta**
In this project, The following tasks are performed

Create a new game with the name of your choice by using the API.
Implement the "Refresh" button (receiving data from the API and parsing the JSON).
Implement the form "Submit" button (sending data to the API).
Use async and await JavaScript features to consume the API.

Kindly review my code and give me a review.
Thanks in Advance

### CODE REVIEW

File: src/leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        scoresList.appendChild(item);
+      }
+    }
+    return true;
+  }
+
+  async fetchApi(url, method, jsonBody = null) {
+    const response = await fetch(url, {
+      headers: {
+        // Accept: 'application/json',
```
Comment (ZawadiSumba66) [inline src/leaderboard.js]: - [ ] Kindly let's remove commented code from the codebase to make it look cleaner and neater.

File: src/leaderboard.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+export default class LeaderBoard {
```
Comment (ZawadiSumba66) [inline src/leaderboard.js:1]: - [ ] In the [javascript requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) point 4, you are encouraged to use ES6 syntax. In this case the arrow functions are highly recommended for all your functions in the codebase. The arrow function is really awesome and makes your code more readable, more structured, and look like modern code. Take a look at this [material](https://stackoverflow.com/questions/42964102/syntax-for-an-async-arrow-function) on how to go about it.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @tta2yta 👋 , 

Good job so far!
However, there are some issues that you still need to work on to go to the next step but you are almost there! 💪 💪 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @tta2yta 👋🏾  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**